sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultRoleAssignmentProperties.cs

```diff
@@ -3,10 +3,10 @@
 
 using Azure.Core;
 
-namespace Azure.Security.KeyVault.Administration.Models
+namespace Azure.Security.KeyVault.Administration
 {
     /// <inheritdoc/>>
     [CodeGenModel("RoleAssignmentProperties")]
-    public partial class KeyVaultRoleAssignmentProperties
+    internal partial class KeyVaultRoleAssignmentProperties
```

Comment:
Could this be a struct? We do that throughout the other libraries.

Reply:
It probably could be, but is it worth changing for a generated class? I've never seen us change something from class to struct.

Reply:
I did in Search because we didn't expose it and it got wrapped anyway.

Reply:
That said, it's a minor improvement for something that shouldn't be a hot code path.